### PR TITLE
krisp 2.33.5

### PR DESCRIPTION
**Pull request by qeeqez**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Comment by qeeqez** [comment]
### There's wrong livecheck in a cask. 
It's pointing to a blog "whats new" https://whatsnew.krisp.ai
But this blog doesn't have a post for each version. 

I can't find another correct page for release notes. 
So maybe we should remove this livecheck at all.

**Comment by krehel** [comment]
Upstream does point to this as being the release notes, but if it's slow to update we can look to something else.

We might be able to use the `headers[location]` with this.  I'll make a test fix.

**Comment by krehel** [comment]
@samford - updated the livecheck here in this, would like your eyes on it if possible to make sure this is appropriate.

Thank you!

File: Casks/k/krisp.rb [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
-    url "https://whatsnew.krisp.ai"
-    regex(/krisp\sv?(\d+(?:\.\d+)+)/i)
+    url "https://download.krisp.ai/mac?package=package_#{livecheck_arch}"
+    regex(%r{/Krisp[._-](\d+(?:\.\d+)+)[._-]#{arch}\.pkg}i)
+    strategy :header_match do |headers, regex|
+      match = headers["location"]&.match(regex)
+      next if match.blank?
+
+      match[1]
+    end
```
Comment (samford) [inline Casks/k/krisp.rb:22]: `HeaderMatch` checks the `Location` and `Content-Disposition` headers by default, so this `strategy` block isn't necessary (i.e., it replicates the default behavior). I'll push a commit to update this in a moment.

**Review by samford — APPROVED** [review]

File: Casks/k/krisp.rb (hunk 1 above)
Comment (krehel) [reply Casks/k/krisp.rb]: Thank you @samford!

**Review by krehel — APPROVED** [review]

**Pull request merged**